korean/marshmallow.md

```diff
@@ -1,7 +1,9 @@
-어느날 잠에서 깨어보니 당신은 벽이 마시멜로로 되어있는 이상한 방 안에서 잠이 깬 것을 발견했어요.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.